kernel/safe_typing.ml

```diff
@@ -131,6 +132,24 @@ type section_data = {
   rev_reimport : reimport list;
 }
 
+module HandleMap = Opaqueproof.HandleMap
+
+(** We rely on uniqueness of pointers to provide a simple implementation of
+    kernel certificates. For this to work across processes, it assumes that
+    safe environments are marshalled at the same time as their corresponding
+    certificates. *)
+module Nonce :
```

Comment:
I actually didn't know the slang meaning, but not all dictionaries I've looked at actually mention it. Meanwhile it's a very standard crypto concept, so I don't think this will be causing trouble.